src/symbolize/gimli.rs

```diff
@@ -5,7 +5,7 @@
 //! intended to wholesale replace the `libbacktrace.rs` implementation.
```

Comment:
Should this comment be updated? Isn't gimli the default now? (Not necessary to do here, just wondering.)